parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh

```diff
@@ -34,27 +34,22 @@ installDeps() {
     local OSVERSION
     OSVERSION=$(grep DISTRIB_RELEASE /etc/*-release| cut -f 2 -d "=")
     BLOBFUSE_VERSION="1.4.5"
-    BLOBFUSE2_VERSION="2.2.1"
+    # Blobfuse2 has been upgraded to 2.3.0 in upstream, using this version for parity between 22.04 and 24.04
+    BLOBFUSE2_VERSION="2.3.0"
```

Comment:
Doesn't need to be this PR but just early checking.
Now we are only working on the component versions management in manifest.json.
Later when we want to centralize the others, is this BLOBFUSE2_VERSION supposed to move to manifest.json/components.json?

Reply:
very good point, it definitely can, however, like the other components there, this isn't updated as frequently. for eg, this version bump is happening after more than 1 year.

however, with how we are moving things toward a centralized location, I think it makes sense to move it, we should just float it by Andy Zhang and storage team as well since if it is updated, it'd be by them.

Reply:
Makes sense. Once we complete the task, we should definitely let the stakeholders know.

Reply:
or is this only installed with CSE so dynamically ? and it's not cached ?

Reply:
@Devinwong  I have added the component to the wiki for tracking https://msazure.visualstudio.com/CloudNativeCompute/_wiki/wikis/CloudNativeCompute.wiki/678597/Managed-Component-List?anchor=os-packages